[2025-04-09T16:11:09.448+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T16:11:09.475+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.create_raw_schema manual__2025-04-09T16:11:06+00:00 [queued]>
[2025-04-09T16:11:09.489+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.create_raw_schema manual__2025-04-09T16:11:06+00:00 [queued]>
[2025-04-09T16:11:09.490+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-09T16:11:09.664+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): create_raw_schema> on 2025-04-09 16:11:06+00:00
[2025-04-09T16:11:09.674+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=957) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-09T16:11:09.675+0000] {standard_task_runner.py:72} INFO - Started process 963 to run task
[2025-04-09T16:11:09.676+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'create_raw_schema', 'manual__2025-04-09T16:11:06+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmp1abgex6u']
[2025-04-09T16:11:09.677+0000] {standard_task_runner.py:105} INFO - Job 87: Subtask create_raw_schema
[2025-04-09T16:11:09.754+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.create_raw_schema manual__2025-04-09T16:11:06+00:00 [running]> on host 5a8aeda502d0
[2025-04-09T16:11:09.912+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='create_raw_schema' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T16:11:06+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T16:11:06+00:00'
[2025-04-09T16:11:09.913+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-09T16:11:09.927+0000] {sql.py:266} INFO - Executing: CREATE SCHEMA IF NOT EXISTS driven_raw;
[2025-04-09T16:11:09.939+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-09T16:11:09.960+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-09T16:11:09.966+0000] {sql.py:509} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS driven_raw;, parameters: None
[2025-04-09T16:11:09.981+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-09T16:11:09.982+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=create_raw_schema, run_id=manual__2025-04-09T16:11:06+00:00, execution_date=20250409T161106, start_date=20250409T161109, end_date=20250409T161109
[2025-04-09T16:11:10.094+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T16:11:10.122+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-09T16:11:10.166+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
